.travis/maybe-publish-gh-pages.sh

```diff
@@ -1,5 +1,13 @@
 #!/bin/bash
 
+source $(npm root -g)/pr-bumper/.travis/is-bump-commit.sh
+
+if isBumpCommit
+then
+  echo "Skipping gh-pages publish for version bump commit"
```

Comment:
I would think, with this being called in `after_success` after the version is bumped, this would always enter here and skip the publish.